scripts/consumer_apps_builder.py

```diff
@@ -38,10 +38,16 @@ def checkout_or_update_static_resources_repo():
     os.chdir('../')
 
 
-def build_apks_from_resources(build_type):
+def build_apks_from_resources(build_type, subset_of_apps_to_build):
     for (app_dir_name, sub_dir_list, files_list) in os.walk(PATH_TO_STATIC_RESOURCES_DIR):
         if '.git' not in app_dir_name and app_dir_name != PATH_TO_STATIC_RESOURCES_DIR:
-            build_apk_from_directory_contents(app_dir_name, files_list, build_type)
+            if subset_of_apps_to_build is None or in_list(subset_of_apps_to_build, app_dir_name):
```

Comment:
I don't see where subset_of_apps_to_build is ever used besides to check if its present/valid or not - not really clear what its supposed to be doing/what values it can take?